src/app/components/header/header.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Title } from '@angular/platform-browser';
+import { AuthService } from 'src/app/services/auth/auth.service';
+import { Router } from '@angular/router';
+import { Observable } from 'rxjs';
+
+
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss']
+})
+export class HeaderComponent implements OnInit {
+  @Input() title;
+
+  public isAuth: boolean;
+
+  constructor(private authService: AuthService, private router: Router) { }
+
+  ngOnInit() {
+    this.authService.isAuth.subscribe(isAuth => this.isAuth = isAuth);
```

Comment:
yes, that's one of the ways to do it. or just this.isAuth = this,authservice.isAuth